paymentsheet/src/test/java/com/stripe/android/customersheet/utils/CustomerSheetTestHelper.kt

```diff
@@ -99,12 +100,20 @@ internal object CustomerSheetTestHelper {
             createModifiableEditPaymentMethodViewInteractorFactory(),
         errorReporter: ErrorReporter = FakeErrorReporter(),
     ): CustomerSheetViewModel {
+        val dataSourceProvider = CompletableDeferred(
+            CustomerAdapterDataSource(
+                customerAdapter = customerAdapter
+            )
+        )
+
         return CustomerSheetViewModel(
             application = application,
             workContext = workContext,
             originalPaymentSelection = savedPaymentSelection,
             paymentConfigurationProvider = { paymentConfiguration },
-            customerAdapterProvider = CompletableDeferred(customerAdapter),
+            paymentMethodDataSourceProvider = dataSourceProvider,
+            intentDataSourceProvider = dataSourceProvider,
+            savedSelectionDataSourceProvider = dataSourceProvider,
```

Comment:
I have fakes made for each of these in a follow-up PR though can merge into this PR if we feel that is appropriate.

Reply:
doing this in a follow up sgtm!